spec/helpers/blacklight/url_helper_behavior_spec.rb

```diff
@@ -241,7 +241,7 @@
     end
 
     it "converts the counter parameter into a data- attribute" do
-      allow(helper).to receive(:track_test_path).with(hash_including(id: '123456', counter: 5)).and_return('tracking url')
+      allow(helper).to receive(:track_test_path).with(hash_including(id: an_instance_of(SolrDocument), counter: 5)).and_return('tracking url')
```

Comment:
I'm not sure it's important, but this could also be `have_attributes(id: '123456')` if we want to make sure we're getting the expected object?

Reply:
Cool, I hadn't seen `have_attributes` before.